drivers/serial/uart_nrfx_uarte.c

```diff
@@ -11,6 +11,28 @@
 #include <uart.h>
 #include <hal/nrf_gpio.h>
 #include <hal/nrf_uarte.h>
+#include <nrfx_timer.h>
+#include <misc/util.h>
+#include <logging/log.h>
+
+/* Generalize PPI or DPPI channel management */
+#if CONFIG_HAS_HW_NRF_PPI
```

Comment:
`#if defined(CONFIG_HAS_HW_NRF_PPI)` here
and
`#elif defined(CONFIG_HAS_HW_NRF_DPPI)` below.